src/mmu.rs

```diff
@@ -912,7 +920,7 @@ impl PageTable {
         if !self.is_mapped(va) {
             return Err(Error::BadPointer);
         }
-        let ptr = core::ptr::from_exposed_addr_mut::<()>(va);
+        let ptr = core::ptr::without_provenance_mut::<()>(va);
```

Comment:
All the other `core::ptr` changes are going from `invalid[_mut]` to `without_provenance[_mut]` except for this one. Should this just have been using `invalid_mut` to start with?

Reply:
Whoops! Thanks for catching that. No, this one should have stayed as `from_exposed_addr`. It got changed because initially I thought to change all of the `invalid`s to `from_exposed_addr`, and changed a couple, but then thought better of it (and in particular that `without_provenance` sort of fit better semantically) and went to change them again, and accidentally swept this one up as well.